tools/cldr-apps/js/src/esm/cldrSurvey.js

```diff
@@ -562,7 +564,7 @@ function updateStatusLoadHandler(json) {
   if (wasBusted == false && json.status.isSetup && loadOnOk != null) {
     window.location.replace(loadOnOk);
   } else {
-    setTimeout(updateStatus, timerSpeed);
+    statusTimeout = setTimeout(updateStatus, timerSpeed);
```

Comment:
should this also clear the old timeout? or is this only called the first time.

Reply:
this is in updateStatus, which only gets called in response to the timer (except the first time), and it sets

  statusTimeout = null;

right at the beginning -- I think clearTimeout would be redundant there since it's called in response to the timer itself